test/controllers/sessions_res_controller_test.rb

```diff
@@ -64,7 +64,7 @@ def test_signin_creation_notsuccess5
   end
 
   def test_signin_creation_notsuccess6
-    value = {identity: "Owner", email: @worker.email, password: "Bbcdef@1"}
+    value = {identity: "Owner", email: @worker.email, password: Faker::Internet.password(min_length: 8,  mix_case: true, special_characters: true)+ Faker::Internet.password}
```

Comment:
please create a private method for payload and pass the values

Reply:
changed